tune/tune.c

```diff
@@ -17,7 +17,8 @@
 static void usage(void)
 {
 	fprintf(stderr, "Usage: tune.exfat\n");
-	fprintf(stderr, "\t-l | --label         Set/Get volume label\n");
+	fprintf(stderr, "\t-l | --print-label   Print volume label\n");
+	fprintf(stderr, "\t-L | --set-label     Print volume label\n");
```

Comment:
Wrong description?

Reply:
Oops, Will fix it. Thanks for your review!